src/App.css

```diff
@@ -1,3 +1,10 @@
+html {
+  width: 100%;
+  height: 100%;
+  margin: 0px;
+  padding: 0px;
+  overflow-x: hidden;
+}
```

Comment:
newlines....please proofread these in all your PRs

Reply:
sure thanks